.buildkite/longruns/pipeline.yml

```diff
@@ -2,7 +2,7 @@ env:
   JULIA_VERSION: "1.9.3"
   MPI_IMPL: "openmpi"
   OPENMPI_VERSION: "4.1.1"
-  CUDA_VERSION: "11.2"
+  CUDA_VERSION: "12.2"
```

Comment:
I've moved away from specifying everything in env variables. This will probably require a bigger change

Reply:
I can open an issue for this in the coupler. Is it just openmpi and cuda that should be removed from the env vars? It looks like Atmos still has a fair amount of env vars: https://github.com/CliMA/ClimaAtmos.jl/blob/main/.buildkite/pipeline.yml#L6-L17

Reply:
i made some changes to how the configuration works, to make it a little more explicit (basically you have to explicitly list the modules you want to load). I can help you with it tomorrow